### PR TITLE
Installer cleanup and  Migrate installer configuration to private repo

### PR DESCRIPTION
**Pull request by SoorajSNair**
CAPI-1035 and CAPI-1040 : 
Installer cleanup - Phase 1 and  Migrated jazz-installer-vars.json to bitbucket repo

@Feroz-Shaikh @suryajak  Please Code Review.

### CODE REVIEW

File: installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{
```
Comment (suryajak) [inline installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json]: Lets standardize on naming convention. Use all CAPS for objects and their keys. For keys within a node object, lets remove redundancy. For example "API_BUILD" : {	"API_BUILD_URI": "", "API_BUILD_URI_DEV": "" }

lets change it to "API" : {	"BUILD_URI": "", "BUILD_URI_DEV": "" }

Please take care of this in every place.

File: installscripts/jazz-terraform-unix-noinstances/provisioners.tf [hunk 2]
```diff
@@ -126,7 +148,33 @@ resource "null_resource" "configureExistingBitbucketServer" {
...
+
+  connection {
+    host = "${lookup(var.jenkinsservermap, "jenkins_public_ip")}"
+    user = "${lookup(var.jenkinsservermap, "jenkins_ssh_login")}"
+    type = "ssh"
+    private_key = "${file("${lookup(var.jenkinsservermap, "jenkins_ssh_key")}")}"
+  }
+    provisioner "remote-exec"{
+    inline = [
+        "mkdir ~/jazz_build_module_tmp ",
```
Comment (suryajak) [inline installscripts/jazz-terraform-unix-noinstances/provisioners.tf]: Why do we need 168, 169, 177?

**Comment by SoorajSNair** [comment]
Made above mentioned changes in the pull request. Please code review

File: installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  "AWS_CREDENTIAL_ID": "",
+  "env_name_prefix": "",
```
Comment (Deepak-Babu) [inline installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json]: correct case as suggested by Surya

File: installscripts/jazz-terraform-unix-noinstances/cloudfront.tf [hunk 4]
```diff
@@ -6,6 +6,10 @@ resource "aws_cloudfront_origin_access_identity" "origin_access_identity" {
   provisioner "local-exec" {
     command = "${var.modifyPropertyFile_cmd} CLOUDFRONT_ORIGIN_ID ${aws_cloudfront_origin_access_identity.origin_access_identity.cloudfront_access_identity_path} ${var.jenkinspropsfile}"
   }
+
+  provisioner "local-exec" {
+    command = "${var.modifyPropertyFile_cmd} CLOUDFRONT_ORIGIN_ID ${aws_cloudfront_origin_access_identity.origin_access_identity.cloudfront_access_identity_path} ${var.jenkinsjsonpropsfile}"
```
Comment (Deepak-Babu) [inline installscripts/jazz-terraform-unix-noinstances/cloudfront.tf]: what is the change here? These lines are the same as the preceeding 3 lines

**Pull request closed without merging**